tests/appsec/iast_packages/test_packages.py

```diff
@@ -109,56 +141,309 @@ def install_latest(self):
+    PackageForTesting(
+        "lxml",
+        "5.2.2",
+        "",
+        "",
+        "",
+        test_e2e=False,
+        import_name="lxml.etree",
+        import_module_to_validate="lxml.doctestcompare",
+    ),
+    PackageForTesting(
+        "more-itertools",
+        "10.2.0",
+        "",
+        "",
+        "",
+        test_e2e=False,
+        import_name="more_itertools",
+        import_module_to_validate="more_itertools.more",
+    ),
+    PackageForTesting(
+        "multidict", "6.0.5", "", "", "", test_e2e=False, import_module_to_validate="multidict._multidict_py"
     ),
-    PackageForTesting("idna", "3.6", "xn--eckwd4c7c.xn--zckzah", "ドメイン.テスト", "xn--eckwd4c7c.xn--zckzah"),
     # Python 3.12 fails in all steps with "import error" when import numpy
-    PackageForTesting("numpy", "1.24.4", "9 8 7 6 5 4 3", [3, 4, 5, 6, 7, 8, 9], 5, skip_python_version=[(3, 12)]),
+    PackageForTesting(
+        "numpy",
+        "1.24.4",
+        "9 8 7 6 5 4 3",
+        [3, 4, 5, 6, 7, 8, 9],
+        5,
+        skip_python_version=[(3, 12)],
+        import_module_to_validate="numpy.core._internal",
+    ),
+    PackageForTesting("oauthlib", "3.2.2", "", "", "", test_e2e=False, import_module_to_validate="oauthlib.common"),
+    PackageForTesting("openpyxl", "3.1.2", "", "", "", test_e2e=False, import_module_to_validate="openpyxl.chart.axis"),
+    PackageForTesting(
+        "packaging",
+        "24.0",
+        "",
+        {"is_version_valid": True, "requirement": "example-package>=1.0.0", "specifier": ">=1.0.0", "version": "1.2.3"},
+        "",
+    ),
+    # Pandas dropped Python 3.8 support in pandas>2.0.3
+    PackageForTesting("pandas", "2.2.2", "", "", "", test_e2e=False, skip_python_version=[(3, 8)]),
+    # TODO: Test import fails with AttributeError: 'FormattedValue' object has no attribute 'values'
+    # PackageForTesting("pillow", "10.3.0", "", "", "", test_e2e=False, import_name="PIL.Image"),
+    PackageForTesting(
+        "platformdirs", "4.2.2", "", "", "", test_e2e=False, import_module_to_validate="platformdirs.unix"
+    ),
+    PackageForTesting("pluggy", "1.5.0", "", "", "", test_e2e=False, import_module_to_validate="pluggy._hooks"),
+    PackageForTesting(
+        "pyasn1",
+        "0.6.0",
+        "Bruce Dickinson",
+        {"decoded_age": 65, "decoded_name": "Bruce Dickinson"},
+        "",
+        import_module_to_validate="pyasn1.codec.native.decoder",
+    ),
+    PackageForTesting("pycparser", "2.22", "", "", ""),
+    PackageForTesting("pydantic", "2.7.1", "", "", "", test_e2e=False),
+    PackageForTesting(
+        "pydantic-core",
+        "2.18.2",
+        "",
+        "",
+        "",
+        test_e2e=False,
+        import_name="pydantic_core",
+        import_module_to_validate="pydantic_core.core_schema",
+    ),
+    # TODO: patching Pytest fails: ImportError: cannot import name 'Dir' from '_pytest.main'
+    # PackageForTesting("pytest", "8.2.1", "", "", "", test_e2e=False),
```

Comment:
ditto

Reply:
Oh I actually fixed that issue in this PR but forgot to uncomment that one, will test again with it uncommented and update.